config/standalone.example.toml

```diff
@@ -1,9 +1,14 @@
 node_id = 0
 mode = 'standalone'
-http_addr = '127.0.0.1:4000'
+datanode_mysql_addr = '127.0.0.1:4406'
```

Comment:
Please rebase the latest develop branch, these two options were already removed.

Reply:
I don't have other suggestions, we can merge this PR after fixing this. Thanks a lot.

Reply:
ok, a little wired about this rebase, caz I kept rebasing. 

While, use #624 instead, thank you.

Reply:
close this pr